packages/blocks/src/lib/navigation-map/components/map.svelte

```diff
@@ -103,15 +120,31 @@ let didHoverTooltip = Boolean(
       <Button on:click={toggleTileset}>
         {satellite ? 'Map' : 'Satellite'}
       </Button>
-      <CenterInputs />
-    </div>
-
-    <div class="absolute bottom-12 right-3 z-10">
       <ToggleButtons
         options={['2D', '3D']}
         selected={$view}
         on:input={handleViewSelect}
       />
+      <CenterInputs />
+    </div>
+
+    <div class="absolute bottom-10 right-2 z-10">
+      <Button
+        disabled={!baseGeoPose}
+        on:click={() => {
+          if (isFollowingBase) {
+            stopFollowingBase();
+          } else {
+            startFollowingBase();
+          }
+        }}
```

Comment:
We can do a little less here

```suggestion
        on:click={isFollowingBase ? stopFollowingBase : startFollowingBase}
```

Reply:
🤦🏼  thank you